grails-app/domain/au/org/ala/profile/Attribute.groovy

```diff
@@ -14,6 +14,7 @@ class Attribute implements Comparable<Attribute> {
         root = false
         only = ["text", "title"]
         title component: true
+        text index: "true"
```

Comment:
SHould this be a string or a boolean?

Reply:
Unfortunately, ES plugin uses string representation of boolean value.